phpBB/posting.php

```diff
@@ -44,6 +41,75 @@
 $submit = $request->is_set_post('post') && !$refresh && !$preview;
 $mode		= $request->variable('mode', '');
 
+// We need to choose which of the url parameters we must trust and ignore the others
+	$forum_id = 0;
+	$topic_id = 0;
+	$post_id = 0;
+
+switch ($mode)
+{
+	case 'popup':
+	case 'smilies':
+		$forum_id = $request->variable('f', 0);
+		break;
```

Comment:
`break;` should have the same indentation as `case`.

Reply:
OK